pixi.toml

```diff
@@ -31,9 +31,16 @@ mypy = ">=1.11,<1.12"
 # For detecting cpu cores with pytest-xdist
 psutil = ">=6.0.0,<7"
 # For running tests in parallel, use this instead of regular pytest
+filelock = ">=3.16.0,<4"
 pytest-xdist = ">=3.6.1,<4"
+rich = ">=13.7.1,<14"
+toml = ">=0.10.2,<0.11"
 
 [feature.pytest.tasks]
+test-common-wheels-ci = { cmd = "pytest -n logical tests/wheel_tests/" }
+test-common-wheels-dev = { cmd = "pytest -n logical tests/wheel_tests/", depends-on = [
+  "build",
+] }
 test-integration-ci = "pytest -n logical tests/integration"
 test-integration-dev = { cmd = "pytest -n logical tests/integration", depends-on = [
   "build",
```

Comment:
```suggestion
test-common-wheels-ci = { cmd = "pytest -n logical tests/wheel_tests/" }
test-common-wheels-dev = { depends-on = ["build","test-common-wheels-ci"] }
test-integration-ci = "pytest -n logical tests/integration"
test-integration-dev = { depends-on = [ "build", "test-integration-ci"
```

Reply:
This does work but makes it less ergonomic. As it disables the ability to filter with `-k`, as there is not command, so if it is alright, I'll leave as is :)

Reply:
Okay, fine by me